barretenberg/cpp/src/barretenberg/flavor/flavor.hpp

```diff
@@ -109,6 +109,7 @@ class ProvingKey_ : public PrecomputedPolynomials, public WitnessPolynomials {
     std::vector<uint32_t> recursive_proof_public_input_indices;
     bb::EvaluationDomain<FF> evaluation_domain;
     std::shared_ptr<CommitmentKey_> commitment_key;
+    std::array<Polynomial, 4> sorted_polynomials;
```

Comment:
maybe this should only belong in the ultra and goblin_ultra flavors

Reply:
Yeah can you put it there instead please. There will be code duplication but that's ok